go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/aws/aws-sdk-go v1.35.3
 	github.com/axw/gocov v1.0.0
+	github.com/bytedance/gopkg v0.0.0-20221122125632-68358b8ecec6
```

Comment:
Where do we use it?

Reply:
Thanks a lot, I will clean it in the next commit.